mdviewer.yml

```diff
@@ -0,0 +1,16 @@
+
+---
+services:
+  ocis:
+    depends_on:
+      md-viewer:
+        condition: service_completed_successfully
+
+  md-viewer:
+    image: jankaritechnepal/md-viewer:latest
```

Comment:
make sure docker image is available on https://hub.docker.com/u/jankaritechnepal before merging this PR